### DISCUSSION TITLE
Edit Hostname within WSL

### DISCUSSION THREAD

**Discussion by artoghraee07** [idaholab/moose#21837] [Q&A Getting Started] [answered]
Hello MOOSE users,

I am trying to install MOOSE on the WINDOSW 10 and I am not very familiar with GitHub and computer environment configuration terms.
As a part of installation  on the "Edit Hostname within WSL" it askes to modify the host file. I run the "Notepad "C:\Windows\System32\Drivers\etc\hosts" at it opens a text file. My question is what should I do next?
- should I , in the "#       127.0.0.1       localhost"  line  just replace the 127.0.01 with the result of hostname? 
- or , should I just add this line "127.0.0.1   <the result of hostname>    <---- ADD THAT " at the end of the text file?

Thanks you for the help

**Comment by milljm** [comment, marked as answer]
You want to always append to this file, and not replace other lines already there. You'll want to figure out what your hostname is. That is achieved by running `hostname` in your cmd.exe terminal window:

```bash
Microsoft Windows [Version 10.0.19044.1586]
(c) Microsoft Corporation. All rights reserved.

C:\Users\Jason>hostname
DESKTOP-IA0VD92
```
For me, I would take `DESKTOP-IA0VD92` and append the following line to my hosts file (located at: `C:\Windows\System32\Drivers\etc\hosts`):
```pre
127.0.0.1   DESKTOP-IA0VD92
```
